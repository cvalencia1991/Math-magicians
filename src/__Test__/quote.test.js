import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Quote from '../Pages/quote';

describe('Testing Quote', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(<Quote />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
