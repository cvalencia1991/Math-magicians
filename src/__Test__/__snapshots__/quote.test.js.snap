// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Quote renders correctly 1`] = `
<div
  className="quotestyle"
>
  Mathematics is not about numbers,equations,computations,or algorithms: it is about understandig.-William Paul Thurston
</div>
`;
